Podfile

```diff
@@ -23,6 +23,7 @@ target 'Eatery' do
     pod 'NVActivityIndicatorView'
     pod 'SnapKit'
     pod 'lottie-ios'
+    pod 'Wormholy', :configurations => ['Debug', 'TestFlight']
```

Comment:
what's the difference between the `Debug` and `TestFlight` configurations? based on the `.xcsheme` files in the commit, it looks like the testflight config isn't used?

Reply:
I dug around and found the TestFlight .xcscheme file inside xcuserdata, whereas all the rest are in xcshareddata? I'm not sure what's up with that, but I recreated the scheme and that seemed to have moved it

Reply:
I had to create a TestFlight scheme (instead of just using Debug) because FLEX uses UIWebView which Apple no longer accepts